File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/UserSession+CoreDataProperties.m
//
//  UserSession+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/17/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

#import "UserSession+CoreDataProperties.h"

@implementation UserSession (CoreDataProperties)

+ (NSFetchRequest<UserSession *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"UserSession"];
}

@dynamic access_token;
@dynamic accound_id;
@dynamic currentSession;
@dynamic expires_at;
@dynamic isCurrent;
@dynamic nickname;

@end

File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/UserSession+CoreDataProperties.swift
//
//  UserSession+CoreDataProperties.swift
//
//
//  Created by Pavel Yeshchyk on 4/23/20.
//
//

import Foundation
import CoreData

extension UserSession {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<UserSession> {
        return NSFetchRequest<UserSession>(entityName: "UserSession")
    }

    @NSManaged public var access_token: String?
    @NSManaged public var accound_id: String?
    @NSManaged public var currentSession: NSNumber?
    @NSManaged public var expires_at: NSNumber?
    @NSManaged public var isCurrent: NSNumber?
    @NSManaged public var nickname: String?
}
